src/main/java/org/apache/commons/io/function/IOConsumer.java

```diff
@@ -85,13 +150,25 @@
      */
     void accept(T t) throws IOException;
 
+    /**
+     * Returns this {@code IOConsumer} wrapped inside of a {@link Consumer}
+     * that throws {@link UncheckedIOException} for any {@link IOException}s
+     * that are thrown by this {@code IOConsumer}.
+     *
+     * @return a {@code Consumer} that wraps this {@code IOConsumer}.
+     * @since 2.12.0
+     */
+    default Consumer<T> asConsumer() {
+        return wrap(this);
```

Comment:
I do not think we need an extra method that just delegates to an existing one. What is the added value? Am I missing something here?

Reply:
Hi @garydgregory , my thought process at the time was that the user may want to  override this method with their own implementation of IOConsumer, but after considering your comments, it would be just as easy for the user who requires a custom implementation to provide that themselves externally, and otherwise rely on the static method for wrapping. I will remove this method.

Reply:
Hi @garydgregory , my thought process at the time was that the user may want to override this method with their own implementation of IOConsumer, but after considering your comments, it would be just as easy for the user who requires a custom implementation to provide that themselves externally, and otherwise rely on the static method for wrapping. I will remove this method.